### PR TITLE
Feat/disabled input

### PR DESCRIPTION
**Pull request by jonatanhanson**
## Explain the changes you’ve made

Added functionality and styling for disabled text inputs. 

## How to test the changes?

Swap the application to run storybook
Navigate to input component
There is an example of disabled text input

## Was this feature tested in the following environments?
- [] Storybook on a iOS device/simulator.
- [] Building the Application on a iOS device/simulator.
- [X] Building the Application on a Android device/simulator.

## Screenshots
Screenshot 2021-06-01 at 15 34 32

### CODE REVIEW

**Comment by helsingborgsstad** [comment]
Task linked: [CU-kq9vg3 Icke redigerbara text och nummer fält, ska renderas korrekt i appen](https://app.clickup.com/t/kq9vg3)

File: source/components/atoms/Input/Input.tsx [hunk 1]
```diff
@@ -44,6 +44,12 @@ const StyledTextInput = styled.TextInput<InputProps>`
   color: ${({ theme }) => theme.colors.neutrals[0]};
   ${(props) => (props.center ? 'text-align: center;' : null)}
   ${(props) => (props.hidden ? 'display: none;' : null)}
+  ${({ editable, theme }) =>
+    editable === false &&
```
Comment (TeddyAndersson) [inline source/components/atoms/Input/Input.tsx:48]: Is there any reason for not using the logical not operator ie ```!editable``` ?

**Review by TeddyAndersson — APPROVED** [review]
The change work from poking around in storybook. Left a minor question in the styled-component changes :)

File: source/components/atoms/Input/Input.tsx (hunk 1 above)
Comment (jonatanhanson) [reply source/components/atoms/Input/Input.tsx:48]: No reason ^^ gonna change it and merge

**Pull request merged**